### PR TITLE
[Tyr] v1 changes

### PR DESCRIPTION
**Pull request by mbounabi**
After review:
- Add new resource for /v1/users that inherits from resource /v0/users
- Remove useless parameter 'version' from /equipment_reports 
- Return 404 when no response for /v1/instances
- Add recommendation file for API v1

### CODE REVIEW

**Review by kinnou02 — APPROVED** [review]

File: source/tyr/tyr/resources.py [hunk 1]
```diff
@@ -979,6 +961,42 @@ def delete(self, user_id, version=0):
...
 
+class UserV1(User):
+    def _get_args(self, user_id):
+        parser = super(UserV1, self)._get_args(user_id)
+        parser.add_argument('page', type=int, required=False, default=1)
+        return parser
+
+    def _get_all_users(self, args):
+        del args['disable_geojson']
+        # dict comprehension would be better, but it's not in python 2.6
```
Comment (pbougue) [inline source/tyr/tyr/resources.py]: minor cleanup (not the point of this PR): All is in 2.7, so we can do it now

File: source/tyr/tests/integration/v1_test.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def test_api():
 
 def test_users(create_5_users):
     check_v1_response('users')
-    check_v1_response('users', 'users/{}'.format(create_5_users))
+    check_v1_response('user', 'users/{}'.format(create_5_users))
```
Comment (pbougue) [inline source/tyr/tests/integration/v1_test.py]: So there was an unspotted "bug" before?
Note for a choice: In Jormungandr, it's plural in that case.

**Review by pbougue — APPROVED** [review]
Tastes better to me :melon:

File: source/tyr/tests/integration/v1_test.py (hunk 2 above)
Comment (mbounabi) [reply source/tyr/tests/integration/v1_test.py]: bug or upgrade, which school is it? We agreed with @kinnou02 that the POST request which returns only 1 user should have a key "user" singular. The request GET /users/user_id also returns only 1 user. So... I changed it to "user" singular. Right or wrong? TBD

Comment (mbounabi) [reply source/tyr/tests/integration/v1_test.py]: What we discussed about the singular is for POST and PUT requests. 
For GET, you're right, it's better to leave the key in the plural for homogeneity, even if there's only one result.

File: source/tyr/tyr/resources.py [hunk 3]
```diff
@@ -987,6 +969,43 @@ def delete(self, user_id, version=0):
...
+            'items_per_page': pagination.per_page,
+            'total_items': pagination.total,
+        }
+        if pagination.has_next:
+            pagination_json['next'] = url_for(request.endpoint, page=pagination.next_num)
+        return marshal(pagination.items, user_fields), pagination_json
+
+    def get(self, user_id=None):
+        resp = super(UserV1, self).get(user_id)
+        if type(resp) == tuple:
```
Comment (AurelienLP) [inline source/tyr/tyr/resources.py:997]: I am a little lost... When is it a tuple ?

Comment (mbounabi) [reply source/tyr/tyr/resources.py:997]: That's what's returned above (l.992) if there are several users and it needs pagination. The response received is a tuple of the users list and the pagination object.

Comment (AurelienLP) [reply source/tyr/tyr/resources.py:997]: Hoooo, I was looking in the method get from User.... My bad. Could you add a little comment `# If we need pagination` somewhere around here ? Could help blind people like me :D

**Review by AurelienLP — APPROVED** [review]

**Review by AurelienLP — APPROVED** [review]

**Review by kinnou02 — APPROVED** [review]

**Review by AurelienLP — APPROVED** [review]

**Pull request merged**